docs/content/dagster-cloud/account/managing-users.mdx

```diff
@@ -364,40 +444,170 @@ Dagster Cloud currently includes support for four levels of role-based access co
-      <td className="bg-red-50">N</td>
-      <td className="bg-red-50">N</td>
-      <td className="bg-green-50">Y</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-red-50">❌</td>
+      <td className="bg-green-50">✅</td>
     </tr>
   </tbody>
 </table>
+
+  </TabItem>
+</TabGroup>
+
+### Assigning user roles
+
+With the exception of the **Organization Admin** role, user roles are set on a per-deployment, per-code location basis and enforced both in Dagster Cloud and the GraphQL API. Organization Admins have access to the entire organization, including all [full deployments](/dagster-cloud/developing-testing/managing-deployments), [code locations](/dagster-cloud/developing-testing/code-locations), and [Branch Deployments](/dagster-cloud/developing-testing/branch-deployments).
+
+#### Deployment permissions
+
+All Dagster Cloud accounts have the ability to set permissions at the deployment level. When defined, the selected user role will be used as the default for all code locations contained in the deployment.
+
+For example, if a user is assigned the **Editor** user role for a deployment, they'll also be an **Editor** for all code locations in the deployment:
+
+<Image
+alt="Default Editor user role for prod code locations"
+src="/images/dagster-cloud/user-token-management/editor-permissions.png"
+width={897}
+height={178}
+/>
+
+For Dagster Cloud Enterprise users, default user roles can be overriden at the [code location level](#code-location-permissions). For non-Enterprise users, the user will have the same level of access for all code locations in the deployment.
+
+When defining user roles at the deployment level, keep the following in mind:
+
+- **Changing the user role for the deployment also removes any overrides that have been set at the code location level**. For example, if a user was a deployment **Editor** and is then changed to a **Viewer**, they'll become a **Viewer** for all code locations in the deployment.
+- **Having access to a deployment doesn't automatically grant access to Branch Deployments**. [Branch deployment permissions](#branch-deployment-permissions) must be granted separately.
+
+#### Code location permissions
+
+<Note>Code location permissions are a Dagster Cloud Enterprise feature.</Note>
+
+Dagster Cloud Enterprise users can override the deployment-level user role by setting permissions at the code location level. Overrides can only be performed if:
+
+- The user is not **Organization Admin**, and
+- The current user role is a **Viewer** or **Launcher** deployment level
+
+Generally, overrides only apply if the override is more permissive than the deployment-level user role. For example, if a user is assigned the **Viewer** user role for a deployment, only a more permissive role can be assigned for a code location in the deployment. Specifically, an **Editor** or **Launcher** user role:
```

Comment:
I think we can drop 'generally', it's always true

Reply:
consider:

For example, if a user is assigned the **Viewer** user role for a deployment, only a more permissive role (in this case, **Editor** or **Launcher**),  can be assigned for a code location in the deployment.